upgrading.md

```diff
@@ -0,0 +1,76 @@
+# Upgrading Prodfiler host-agent
+
+The upgrade process varies highly depending on the type of installation.
+
+The Prodfiler UI project menu `Project > Instructions` contains the project-specific data that _may_ be needed to perform an upgrade.
+
+![Instructions menu](./pictures/instructions.png)
+
+Find below a list of upgrade instructions for each supported platform.
+
+### Binary installation
+
+Use the commands provided in the UI in the `Instructions` menu, switching the `Binary` tab.
+
+### Docker container
+
+Use the commands provided in the UI in the `Instructions` menu, switching to the `Docker` tab.
+
+### OS package in .deb format
+
+Use the commands provided in the UI in the `Instructions` menu, switching to the `DEB Package` tab.
+
+### OS package in .rpm format
+
+* Download the latest stable release visiting [the stable release page](https://releases.prodfiler.com/stable/index.html).
+* Run the upgrade without changing any configs
+        
+      sudo rpm --upgrade pf-host-agent-*.rpm
+
+* Check logs, the version of host-agent is displayed at the start of the output
+
+      sudo journalctl -xu pf-host-agent
+
+### Kubernetes
+
+If you installed via Helm following the instructions in the UI, without any customizations to namespace or release name,
+you can upgrade with the following:
+    
+    PRODFILER_VERSION=$(curl -s https://documentation.prodfiler.com/latest-release.md)
+    helm -n prodfiler upgrade pf-host-agent --set version=${PRODFILER_VERSION} \
+      --set "collectionAgentHostPort=data-v2.run.prodfiler.com:443"
+
+If you did apply different settings for your environment, verify them first and then set the new `version` and `
```

Comment:
Can you move the ` to the next line ? GH has an issue and colors the next lines :-)

Reply:
good spot!